src/System Application/App/Azure Blob Services API/src/ABSClientImpl.Codeunit.al

```diff
@@ -29,6 +29,7 @@ codeunit 9051 "ABS Client Impl."
         AppendBlockFromUrlOperationNotSuccessfulErr: Label 'Could not append block from URL %1 on %2.', Comment = '%1 = Source URI; %2 = Blob';
         GetPropertiesOperationNotSuccessfulErr: Label 'Could not get properties for %1.', Comment = '%1 = Blob';
         TagsOperationNotSuccessfulErr: Label 'Could not %1 %2 Tags.', Comment = '%1 = Get/Set, %2 = Service/Blob, ';
+        MetaDataOperationNotSuccessfulErr: Label 'Could not %1 %2 MetaData.', Comment = '%1 = Get/Set, %2 = Service/Blob, ';
```

Comment:
These partial strings are really tough on translators and really should not be used. You only have 4 possible combinations here, so just spell them out, add them all. 

Yes, I can see ALL the other strings were done incorrectly as well... Someone should fix that. 

Also, you can see they created Labels for their hardcoded strings so you are missing GetLbl and SetLbl if you want to follow their pattern (which is very bad anyway).

Reply:
Updated,
Taken your feedback and made 2 different labels for get and set